### PR TITLE
#167190519 Setup Database With Sequelize ORM

### PR DESCRIPTION
**Pull request by chialuka**
#### What does this PR do?
- Setups databases for the project. The PR creates test, development, and production databases

- Configures the project to use Sequelize ORM

- Configures Sequelize to use ES6 syntax

#### Description of Task proposed in this pull request?
- Sequelize by default uses ES5. The PR uses the .sequelize.rc file to configure Sequelize to use ES6 syntax. It also provides PostgreSQL databases for development and production and an SQLite database for testing.

#### How should this be manually tested (Quality Assurance)?
- There are no changes to the API as this is a chore that sets up the database for the project

#### What are the relevant pivotal tracker stories?
- [#167190519](https://www.pivotaltracker.com/story/show/167190519)

#### Any background context you want to add (Operations Impact)?
- Add a database url connection to the .env file for the DB_URL_DEV value to run a database for the development environment. Also, set NODE_ENV to development. 

To run migrations and generate models, `sequelize`, `sequelize-cli`, `pg`, and `pg-hstore` must be installed as dependencies.

#### What I have learned working on this feature
- I have learned how to use Sequelize, sequelize-cli, sqlite and how to use ES6 syntax with the configuration. I also learned how to set and view NODE_ENV using the terminal on a Mac.

### CODE REVIEW

**Review by henryade — DISMISSED** [review]

**Review by JuwonAbiola — DISMISSED** [review]

**Review by ayodejiAA — DISMISSED** [review]
Nice work

File: .sequelizerc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require("@babel/register");
+require("@babel/register");
```
Comment (dinobi) [inline .sequelizerc]: why the duplicate?

File: .sequelizerc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const path = require('path');
+
+module.exports = {
+  'config': path.resolve('server/database/config', 'config.js'),
+  'models-path': path.resolve('server/database/models'),
+  'seeders-path': path.resolve('server/database/seeders'),
```
Comment (dinobi) [inline .sequelizerc]: this reads better: `'seeds-path': path.resolve('server/database/seeds'),`

**Review by dinobi — COMMENTED** [review]
Please update your branch

File: .sequelizerc (hunk 2 above)
Comment (chialuka) [reply .sequelizerc]: Okay. Those folders are generated automatically by Sequelize. I will rename it to seeds

File: .sequelizerc (hunk 1 above)
Comment (chialuka) [reply .sequelizerc]: That's an error. It's been fixed now.

File: __test__/index.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
+import request from 'supertest';
```
Comment (dinobi) [inline __test__/index.test.js]: Why do we still have supertest here?

**Review by dinobi — APPROVED** [review]

**Review by henryade — APPROVED** [review]

**Review by ayodejiAA — APPROVED** [review]

**Pull request merged**